codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+  status:
+    patch:
+      default:
+        target: 80% # Require at least 80% test coverage on new/changed lines
+        threshold: 2% # Allow a small drop in coverage
+        base: auto
```

Comment:
```suggestion
        base: auto

comment:
  layout: "reach,diff,flags,tree"  # Display different coverage views
  behavior: default                # Default PR comment behavior
  require_changes: true             # Only post if coverage changes
  require_base: yes                 # Compare against base branch coverage

github_checks:
  annotations: true  # Enable GitHub Checks Annotations

```

Reply:
Not saying this is the right format, but we should tweak it.

https://docs.codecov.com/docs/changing-your-pr-comment-format

Reply:
https://docs.codecov.com/docs/common-recipe-list

We def want annotations